rabbitmq-operator/src/test/java/com/indeed/operators/rabbitmq/reconciliation/rabbitmq/TestUserReconciler.java

```diff
@@ -158,6 +162,31 @@ public void testReconcile_updatePermissions() {
         assertEquals("newread", capturedPermission.getRead().pattern());
     }
 
+    @Test
+    public void testReconcile_skipUpToDatePermissions() {
```

Comment:
I think we'll be able to do this for entire Users (rather than just Permissions) once #55 is done